ibis/backends/dask/udf.py

```diff
@@ -116,10 +125,25 @@ def lazy_agg(*series: pandas.Series):
         # Depending on the type of operation, lazy_result is a Delayed that
         # could become a dd.Series or a dd.core.Scalar
         if isinstance(op, ops.AnalyticVectorizedUDF):
-            meta = pandas.Series(
-                [], name=args[0].name, dtype=op._output_type.to_dask()
-            )
+            if isinstance(op._output_type, dt.Struct):
+                meta = make_struct_op_meta(op)
+            else:
+                meta = make_meta_series(
+                    dtype=op._output_type.to_dask(), name=args[0].name,
+                )
             result = dd.from_delayed(lazy_result, meta=meta)
+
+            if args[0].known_divisions:
+                # result is going to be a single partitioned thing,
+                # but we need it to match incoming data so we can dd.concat
+                # downstream. However we know that this udf did not change
+                # the index.
+                original_divisions = args[0].divisions
+                result.divisions = (
```

Comment:
I am not sure I understand this correctly. Looks like the result doesn't match input partitions, how does dd.concat work in that case?

Reply:
It ends up aligning them and joining https://github.com/dask/dask/blob/main/dask/dataframe/multi.py#L973. 

The case you can't have in `dd.concat` is a mix of known and unknown divisions.

Reply:
kk might be worthwhile adding some comments but it might not be obvious to reader that is not familiar with Dask behavior of concat when partitioning doesn't exactly align.